contents/data/sql/SQL.md

```diff
@@ -0,0 +1,145 @@
+<frontmatter>
+  title: Introduction to SQL
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to SQL
+
+**Author(s): [Amrut Prabhu](https://github.com/amrut-prabhu)**
+
+Reviewers: [Ronak Lakhotia](https://github.com/RonakLakhotia), [Rahul Rajesh](https://github.com/rrtheonlyone)
+
+## What is SQL?
+
+Databases are integral to any commercial software application, whether it is a web app, desktop app or otherwise.
+As the size of your application grows, you need to look into dedicated database software packages (like SQL software) to store your data. It will not be enough to simply rely on in-memory solutions or a simple text file to store the data. Many companies like Facebook, Google, Adobe, Paytm and Zappos use <tooltip content="Relational DataBase Management System">RDBMS</tooltip> software to fulfil this need.
+
+**Structured Query Language** (**SQL**) is a programming language that is specifically designed for managing (storing, retrieving and manipulating) data in a <tooltip content="relational databases store data in tables">[relational database]({{baseUrl}}/contents/data/databases/databases.html#database-models)</tooltip>. Unlike other languages, SQL doesn't come as a standalone installation. Rather, _SQL Software_ (or _SQL Implementations_) like MySQL, Oracle, SQL Server and PostgreSQL come with a RDBMS that uses the SQL language.
+However, the syntaxes and functions used across these software systems <tooltip content="in terms of whether the syntax is case-sensitive, the format for specifying date and time, what functions are available out-of-the-box">vary</tooltip>.
+
+<box type="tip">
+  Many online resources refer to Microsoft SQL Server as SQL. So, keep that in mind before you start making conclusions about the characteristics and features of SQL, the language.
+</box>
```

Comment:
Are you sure about this? Microsoft SQL Server is sometimes called _Sequel Server_ but I've never seen it referred to as SQL.

Reply:
Hi Prof, you're right. I tried looking for some counter examples, but they're hard to come by.

However, [here](https://www.dnnsoftware.com/docs/developers/setup/set-up-sql.html) is something that I came across while doing my research. It is a little misleading, isn't it?

I've changed this tip box to be more generic rather than just pointing out SQL Server, since I feel the point is still valid

Reply:
Hi Prof, you're right. I tried looking for some counter examples, but they seem hard to come by.

However, [here](https://www.dnnsoftware.com/docs/developers/setup/set-up-sql.html) is something that I came across while doing my research. It is a little misleading, but it's a rare exception.
The previous paragraph does mention that there's no way to "install SQL" as such. So, I've removed the tip box.

Reply:
That is indeed misleading, but most likely an isolated incident. We shouldn't pollute our article due to such rare cases.